### PR TITLE
Cropping kymographs.

### PR DESCRIPTION
**Pull request by JoepVanlier**
**Why?**
Users often want to exclude the beads from downstream analysis. Therefore it is useful if they can actually crop the kymo easily.

This API introduces Kymograph cropping.

There is also a plan for updating the Cas9 notebook with this functionality, but I will make a separate PR for that.

Note: This PR depends on https://github.com/lumicks/pylake/pull/171

### CODE REVIEW

File: lumicks/pylake/kymo.py
Comment (JoepVanlier) [reply lumicks/pylake/kymo.py:290]: Sure. Given that there is no offset in the position we can. I'll swap it around.

File: lumicks/pylake/tests/test_kymo.py
Comment (JoepVanlier) [reply lumicks/pylake/tests/test_kymo.py:467]: Added a test for this :)

File: lumicks/pylake/kymo.py
Comment (rpauszek) [inline lumicks/pylake/kymo.py:49]: should this be an internal attribute maybe? at least until we determine if we really need it?

File: lumicks/pylake/kymo.py
Comment (rpauszek) [inline lumicks/pylake/kymo.py:43]: I think we should be clear about what the offset is relative to. The most recent parent object that was cropped or the actual raw data?

File: lumicks/pylake/kymo.py
Comment (rpauszek) [inline lumicks/pylake/kymo.py:309]: I think for things like this where you just forward the existing factory it might be cleaner to just have these as lambdas down below. It could also make a clear distinction between things that are changed or not if we consistently use lambdas for forwarded values and function definitions for changed

**Review by rpauszek — COMMENTED** [review]
looks pretty good, just a couple comments about clarity. I'll hold off on approval so I can take a look again after #171 gets merged

File: lumicks/pylake/kymo.py
Comment (JoepVanlier) [reply lumicks/pylake/kymo.py:49]: Alright. I've made it private for now.

File: lumicks/pylake/kymo.py
Comment (JoepVanlier) [reply lumicks/pylake/kymo.py:43]: It's the absolute offset. I'll add it to the docstring.

File: lumicks/pylake/kymo.py
Comment (JoepVanlier) [reply lumicks/pylake/kymo.py]: Sure. I'll update it to do that :)

File: lumicks/pylake/kymo.py
Comment (tiagobonetti) [reply lumicks/pylake/kymo.py]: I'm unsure if the docstring changes showed up in the most recent version.

**Review by tiagobonetti — APPROVED** [review]
`goose.honk_by_distance()`

**Pull request merged**
